client/src/components/BecomeInterpreterForm.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Dropdown } from "semantic-ui-react";
+import { Grid } from "semantic-ui-react";
+
+const languageOptions = [
```

Comment:
This is normally not a good practice to hard code this but if you decided to, then it would be good if you store this constant in a different file and import it here (if you need it in the future) you can just import it anywhere.